js/constants/config.js

```diff
@@ -40,5 +40,6 @@ module.exports = {
     authUrl: (userId) => `${vaultHost}/v1/users/${userId}`,
     replacementUrl: adHost
   },
-  braveExtensionId: 'mnojpmjdmbbfmejpflffifhffcmidifd'
+  braveExtensionId: 'mnojpmjdmbbfmejpflffifhffcmidifd',
+  PDFJSExtensionId: 'ckglimbbooghmggmkikhbolcfmgniemo'
```

Comment:
appears that `chrome.extension.getURL` is returning an arbitrary value in https://github.com/brave/browser-laptop/blob/06a2aa180a54bc4dabafd201203dd75b8748c9f8/app/extensions/pdfjs/extension-router.js#L21 instead of `chrome-extension://<extension_id>`. this does not seem to be an issue in chrome. @bridiver ideas?